### PR TITLE
Added support for displaying GPU memory in the chart

### PR DESCRIPTION
**Pull request by peci1**

### CODE REVIEW

File: system-monitor@paradoxxx.zero.gmail.com/extension.js [hunk 1]
```diff
@@ -2158,15 +2158,19 @@ const Gpu = class SystemMonitor_Gpu extends ElementBase {
...
+            // we subtract percentage from memory because we do not want memory to be 
+            // "accumulated" in the chart with utilization; these two measures should be 
+            // independent
+            this.vals = [this.percentage, this.mem / this.total * 100 - this.percentage];
+            this.tip_vals = [Math.round(this.vals[0]), this.mem];
         }
-        this.text_items[0].text = this.tip_vals.toString();
-        this.menu_items[0].text = this.tip_vals.toLocaleString();
+        this.text_items[0].text = this.tip_vals[0].toString();
+        this.menu_items[0].text = this.tip_vals[0].toLocaleString();
```
Comment (DevDef) [inline system-monitor@paradoxxx.zero.gmail.com/extension.js:2173]: The toLocaleString() method needs the locale as argument now:
this.menu_items[0].text = this.tip_vals[0].toLocaleString(Locale);

File: system-monitor@paradoxxx.zero.gmail.com/schemas/org.gnome.shell.extensions.system-monitor.gschema.xml [hunk 2]
```diff
@@ -398,6 +398,10 @@
     </key>
     <key name="gpu-used-color" type="s">
       <default>'#00b35b'</default>
+      <summary>Color of program gpu usage in the chart</summary>
```
Comment (DevDef) [inline system-monitor@paradoxxx.zero.gmail.com/schemas/org.gnome.shell.extensions.system-monitor.gschema.xml:401]: Please use GPU in capital letters.

**Review by DevDef — CHANGES_REQUESTED** [review]
Please submit a new PR.

**Comment by peci1** [comment]
> Please submit a new PR.

Why?

File: system-monitor@paradoxxx.zero.gmail.com/extension.js (hunk 1 above)
Comment (peci1) [reply system-monitor@paradoxxx.zero.gmail.com/extension.js]: resolved while merging with master

**Comment by DevDef** [comment]
> > Please submit a new PR.
> 
> Why?

You’re right, it’s not needed if you update this one. ;-)

File: system-monitor@paradoxxx.zero.gmail.com/schemas/org.gnome.shell.extensions.system-monitor.gschema.xml (hunk 2 above)
Comment (peci1) [reply system-monitor@paradoxxx.zero.gmail.com/schemas/org.gnome.shell.extensions.system-monitor.gschema.xml]: Fixed capitalization of GPU. I noticed there were other places where GPU had other case, so I fixed them all.

**Comment by peci1** [comment]
I updated with master and resolved both your comments.

**Review by DevDef — APPROVED** [review]

**Comment by storopoli** [comment]
Is this going to be merged soon?

**Pull request merged**